grading/ssh-sudoer/bin/archive

```diff
@@ -0,0 +1,42 @@
+#!/bin/python3
```

Comment:
I guess this file isn't even modified compared to the original one. It does not need to be defined here. Either you rely on a hook system, with hooks file stored in a specific folder, that you add on top on top of the base container (I think this is the cleanest way), either you perform download of the original files while the container is built, in the ``Dockerfile``.